BenchmarkTool/Program.cs

```diff
@@ -53,71 +112,289 @@ static async Task Main(string[] args)
             }
         }
 
-        private static Task<List<QueryStatusWrite>> RunQueryTask(ClientWrite client)
+
+        private async static Task<QueryStatusWrite> RunIngestionTask(ClientWrite client)
         {
-            return client.RunQuery();
+            return await client.RunIngestion(_TestRetryIteration);
```

Comment:
`return client.RunIngestion(...);` and no `async`

Reply:
same as above, i did it during the code audit - if not specified why removing it, i would leave it.